frontend/src/metabase/lib/cron.ts

```diff
@@ -64,3 +67,26 @@ export function explainCronExpression(cronExpression: string) {
     use24HourTimeFormat: has24HourModeSetting(),
   });
 }
+
+function lowerCaseFirstLetter(str: string) {
+  return str.charAt(0).toLowerCase() + str.slice(1);
+}
```

Comment:
This might not work well for German

Reply:
why so, just curious? that is existing functionality, I just moved it to another file.